packages/contractkit/src/utils/rpc-caller.ts

```diff
@@ -110,3 +128,13 @@ export class DefaultRpcCaller implements RpcCaller {
     }
   }
 }
+
+export interface TelemetryHandler {
+  handleEvent: (path: string[], measurements: { duration: number }, metadata: any) => void
+}
+
+export class DebugTelemetryHandler implements TelemetryHandler {
+  handleEvent(path: string[], measurements: { duration: number }, metadata: any) {
```

Comment:
I would add some documentation here... 
What does this is supposed to mean... ? path, measurements? metadata? 

For me; it's something no one else would know what to do with this or what is for

Reply:
Thanks for the comments. This wasn't mean to be an actual PR but just as a way to start the conversation. Would certainly properly type and document the actual payload. Right now it's just to measure how long calls take